modules/monitoring-granting-users-permission-to-monitor-user-defined-projects.adoc

```diff
@@ -13,21 +13,31 @@ You can also grant developers and other users different permissions:
+|`user-workload-monitoring-config-edit` |This role is given in the `openshift-user-workload-monitoring` project. Users with this role can edit the `user-workload-monitoring-config` `ConfigMap` object to configure Prometheus, Prometheus Operator, Alertmanager, and Thanos Ruler for user-defined workload monitoring.
 
-|`monitoring-rules-edit` | Users with this cluster role can create, modify, and delete `PrometheusRule` custom resources for a user-defined project. They can also create and silence alerts in the *Developer* perspective of the {product-title} web console.
+|`monitoring-alertmanager-api-reader` |Users with this role have read access to the Alertmanager API for monitoring of user-defined projects. They can also view alert silences in the *Developer* perspective of the {product-title} web console.
 
-|`monitoring-edit` | Users with this cluster role have the same privileges as users with the `monitoring-rules-edit` cluster role. Additionally, users can create, modify, and delete `ServiceMonitor` and `PodMonitor` resources to scrape metrics from services and pods.
+|`monitoring-alertmanager-api-writer` |Users with this role have read and write access to the Alertmanager API for monitoring of user-defined projects. They can also manage alert silences in the *Developer* perspective of the {product-title} web console.
+|===
+
+.Monitoring cluster roles
+|===
+|Cluster role name |Description
+
+|`monitoring-rules-view` |Users with this cluster role have read access to `PrometheusRule` custom resources (CRs) for user-defined projects. They can also view the alerts in the *Developer* perspective of the {product-title} web console.
+
+|`monitoring-rules-edit` |Users with this cluster role can create, modify, and delete `PrometheusRule` CRs for user-defined projects. They can also create and silence alerts in the *Developer* perspective of the {product-title} web console.
 
-|`user-workload-monitoring-config-edit` | This role is given in the `openshift-user-workload-monitoring` project. Users with this role can edit the `user-workload-monitoring-config` `ConfigMap` object to configure Prometheus, Prometheus Operator, Alertmanager, and Thanos Ruler for user-defined workload monitoring.
+|`monitoring-edit` |Users with this cluster role have the same privileges as users with the `monitoring-rules-edit` cluster role. Additionally, users can create, modify, and delete `ServiceMonitor` and `PodMonitor` resources to scrape metrics from services and pods.
 
-|`alert-routing-edit` | Users with this cluster role can create, update, and delete `AlertmanagerConfig` custom resources for a user-defined project.
+|`alert-routing-edit` |Users with this cluster role can create, update, and delete `AlertmanagerConfig` CRs for user-defined projects.
```

Comment:
```suggestion
|`alert-routing-edit` |Users with this cluster role can create, update, and delete `AlertmanagerConfig` custom resources for user-defined projects.
```

Reply:
Because I already used "custom resources (CRs)" for the `monitoring-rules-view` row, based on the writing rules I should only mention CRs from now on. D you think it can be problematic here? Or it is okay if I keep it as it is?